### DISCUSSION TITLE
Cursor frequently misaligned in codespace IDE.

### DISCUSSION THREAD

**Discussion by umdieweld** [community/community#43531] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Hello all! 

My cursor will frequently become misaligned in codespace. Sometimes doing a full rebuild works, sometimes opening different extensions works, sometimes nothing works. Please see the attached images to see what I'm referring to. I started typing "lmao" from where the cursor was located in the first picture, and in the second picture the cursor is located as far right on the line as it can be before going to the next line. [https://imgur.com/a/1j6T3lt](url)

Anyone got any ideas? I'd prefer to not have to do a full rebuild every time this happens, which is multiple times an hour if I'm tabbing away and back.

**Comment by umdieweld** [comment]
Bump.

**Comment by DatDerpyDude81** [comment]
Have you tried tchanging your screen size and messing around with the zoom?

**Reply by EeliV** [reply]
That worked!

**Comment by cwndrws** [comment]
Hi! Thanks for writing in. I have seen some VS Code extensions cause issues with cursors like this. You could try disabling extensions to alleviate the issue. If you've disabled all extensions and are still seeing the issue, please reply here to let us know that you're still having an issue.

**Reply by brnmcdnld720** [reply]
That worked for me, I was able to disable and then re-enable and it continued fine. I did not extensively test different use cases with this.

**Comment by spenserblack** [comment]
I've noticed this most frequently on a Chromebook (using Chrome, obviously :laughing:) for some reason. Not sure what the exact cause could be. The resolution, maybe?

**Comment by adamlacombe** [comment]
Possible solution: https://github.com/microsoft/vscode/issues/160702#issuecomment-1262870646
